src/components/login/Login.js

```diff
@@ -28,7 +28,7 @@ const Login = () => {
                 name: e.detail.user.name, //user trait
                 email: e.detail.user.email, //user trait
               });
-              navigate("/");
+              navigate(-2);
```

Comment:
can we change the method here? 
Going back 2 steps may result in wrong experience (if I'm not coming from the unauthenticated checkout flow).  
For example-
1. go to cart
2. go to home
3. login
This will result in user going to cart instead of home

We should probably add query param to state where we should go back after successful login...